pkg/provision/ubuntu.go

```diff
@@ -75,12 +75,9 @@ func (p *UbuntuProvisioner) GenerateDockerOptions(dockerPort int) (*provision.Do
 	engineConfigTmpl := `[Unit]
 Description=Docker Application Container Engine
 Documentation=https://docs.docker.com
-BindsTo=containerd.service
-After=network-online.target firewalld.service containerd.service
-Wants=network-online.target
+After=network-online.target nss-lookup.target docker.socket firewalld.service containerd.service time-set.target
+Wants=network-online.target containerd.service
 Requires=docker.socket
-StartLimitBurst=3
```

Comment:
I think we should keep  these because this will prevent minikube's docker to restart too many times on no so powerful 
systems 


StartLimitBurst=3
→ The service can be restarted up to 3 times within the time window defined by StartLimitIntervalSec.

StartLimitIntervalSec=60
→ The time window is 60 seconds. If the service fails and restarts more than 3 times in 60 seconds, systemd will stop trying to restart it automatically.